bin/utils.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @typedef { import("../types/plugins").SanityBuildPluginEntity } SanityBuildPluginEntity
+ * @typedef { import("../types/plugins").SanityPluginLookup } SanityPluginLookup
+ * @typedef { import("../types/plugins").BuildPluginEntity } BuildPluginEntity
+ */
+
+/**
+ *
+ * @param {SanityBuildPluginEntity} plugin
+ *
+ * @returns {BuildPluginEntity}
+ */
+const convertCmsChangesToRepoPlugin = (plugin) => {
+  const { compatibility, description, packageName, status: rawStatus, repoUrl: repo, title: name, version } = plugin
+  /**
+   * @type {BuildPluginEntity['status']}
+   */
+  const status = rawStatus === 'deactivated' ? rawStatus.toUpperCase() : undefined
+
+  return {
+    name,
+    package: packageName,
+    description,
+    repo,
+    version,
+    status,
+    compatibility,
+  }
+}
+
+const stripNullifiedFields = (plugin) => {
+  // If plugin status is undefined, it means the plugin is active, so the field is omitted
+  if (!plugin.status) {
+    // eslint-disable-next-line no-param-reassign
+    delete plugin.status
```

Comment:
First PR i've ever seen with `delete` used, I'm proud

Reply:
I could have maybe done this another way, but we don't care about immutability in this scenario.